packages/docker-engine/daemon-nvidia-json

```diff
@@ -25,5 +25,9 @@ std = { version = "v1", helpers = ["join_array"] }
   {{/if}}
   {{/each}}
   {{/if}}
-  "selinux-enabled": true
+  "selinux-enabled": true,
+  "features": {
+    "cdi": true
+  },
+  "cdi-spec-dirs": ["/etc/cdi/"]
```

Comment:
Question: is there a reason that we ignore `/var/run/cdi`? Is this folder not applicable on Bottlerocket?

Reply:
For ECS, yeah, there is no need to read CDI specs from `/var/run/cdi` as the only CDI specs provider will be `nvidia-ctk`.